### PR TITLE
Delete missing imports, remove frontend_views folder, fix bugs (Closes #1090)

### LINKED ISSUE #1090: Update and move frontend_views
The usage of the `frontent_views` folder is not compatible with the Django convention.

### Tasks

- [x] Reformat `frontend_views/views.py`
    - [x] If it is possible, use f-Strings
    - [x] Fix variable names. The name of the variables should be self-explanatory.
           **Examples of bad naming:**
https://github.com/CiviWiki/OpenCiviWiki/blob/d7f24fad7c0a2565da3bf2cd54e89a847d7479dd/project/frontend_views/views.py#L21
https://github.com/CiviWiki/OpenCiviWiki/blob/d7f24fad7c0a2565da3bf2cd54e89a847d7479dd/project/frontend_views/views.py#L100

- [x] `frontend_views/views.py` -> `accounts/views.py`
     - [x] [user_profile](https://github.com/CiviWiki/OpenCiviWiki/blob/d7f24fad7c0a2565da3bf2cd54e89a847d7479dd/project/frontend_views/views.py#L61)
     - [x] Related URLs (`frontend_views/urls.py` -> `accounts/urls/urls.py`)

 - [x] `frontend_views/views.py` -> `threads/views.py`
     - [x] [base_view](https://github.com/CiviWiki/OpenCiviWiki/blob/d7f24fad7c0a2565da3bf2cd54e89a847d7479dd/project/frontend_views/views.py#L17)
     - [x] [issue_thread](https://github.com/CiviWiki/OpenCiviWiki/blob/d7f24fad7c0a2565da3bf2cd54e89a847d7479dd/project/frontend_views/views.py#L95)
     - [x] [civi2csv](https://github.com/CiviWiki/OpenCiviWiki/blob/d7f24fad7c0a2565da3bf2cd54e89a847d7479dd/project/frontend_views/views.py#L189)
     - [x] Related URLs (`frontend_views/urls.py` -> `threads/urls.py`)

- [x] Discuss which app should include below:
    - [x] [create_group](https://github.com/CiviWiki/OpenCiviWiki/blob/d7f24fad7c0a2565da3bf2cd54e89a847d7479dd/project/frontend_views/views.py#L161)
    - [x] [declaration](https://github.com/CiviWiki/OpenCiviWiki/blob/d7f24fad7c0a2565da3bf2cd54e89a847d7479dd/project/frontend_views/views.py#L165)
    - [x] [landing_view](https://github.com/CiviWiki/OpenCiviWiki/blob/d7f24fad7c0a2565da3bf2cd54e89a847d7479dd/project/frontend_views/views.py#L169)
    - [x] [how_it_works_view](https://github.com/CiviWiki/OpenCiviWiki/blob/d7f24fad7c0a2565da3bf2cd54e89a847d7479dd/project/frontend_views/views.py#L173)
    - [x] [about_view](https://github.com/CiviWiki/OpenCiviWiki/blob/d7f24fad7c0a2565da3bf2cd54e89a847d7479dd/project/frontend_views/views.py#L177)
    - [x] [support_us_view](https://github.com/CiviWiki/OpenCiviWiki/blob/d7f24fad7c0a2565da3bf2cd54e89a847d7479dd/project/frontend_views/views.py#L181)
    - [x] Except for `create_group`, it might be good to use [TemplateView](https://docs.djangoproject.com/en/3.2/ref/class-based-views/base/#templateview) for rest.
    - [x] Related URLs
    - [x] A good reference would be great to follow the Django convention about the views above. For example, `/about/` page does not belong to any model. We should decide where to put it. `thread/views.py` or a new file `core/views.py`?
  
- [x] Refactor for all the usages above. (including all `views.py` files and JS files)

- [x] Remove `frontend_views` folder

Since this is a long task and it requires some knowledge about the project. @AbhijithGanesh, I am assigning this issue.
Please start this issue after merging PR #1088 and #1089.

### PR DESCRIPTION
**Pull request by gorkemarslan**
Closes #1090 

I fix these issues that I can detect:
- Apply changes in PR #1116. In the current branch, some of these changes are gone somehow.
- Delete `threads/view` folder and move all content to `threads/views.py` file
- Remove `frontend_views` folder
- Delete unused and missing imports
- Add flake8 escape for django signals

### CODE REVIEW

**Comment by AbhijithGanesh** [comment]
@gorkemarslan I think flake8 when added to workflows should check unresolved imports, can this be checked?

**Comment by gorkemarslan** [comment]
@AbhijithGanesh, yes definitely! Actually, I don't know why unused imports weren't detected by `pep8speaks`.  And there are missing imports, i.e. some errors like there is no such module. Tests are supposed to detect them but, apparently, they failed.

File: project/accounts/apps.py [hunk 1]
```diff
@@ -4,5 +4,6 @@
 class AccountsConfig(AppConfig):
     name = 'accounts'
 
+    # flake8: noqa: F401
```
Comment (AbhijithGanesh) [inline project/accounts/apps.py:7]: Should this be a necessary comment?

**Review by AbhijithGanesh — COMMENTED** [review]
Please add the init files, it might be required in case of imports

File: project/accounts/apps.py (hunk 1 above)
Comment (gorkemarslan) [reply project/accounts/apps.py:7]: Yes. In order to use signals, we need to make this change in `apps.py`:
https://github.com/CiviWiki/OpenCiviWiki/blob/364b9b1fe85b7610b25d97454342d9c80f5bdb0f/project/accounts/apps.py#L4-L8

It seems unused import but it is how we use signals.

`flake8` can perceive it as an unused import but we intentionally used it. To escape `flake8` warning, we need to put this escaping comment so flake8 won't warn us about `F401` issues for this method. This is simple step. I just want to add it and specify in the PR message

**Comment by gorkemarslan** [comment]
I don't remove only the `__init__.py` file, I also remove the views folder which has an `__init__.py` file.

Could you check the current [view](https://github.com/CiviWiki/OpenCiviWiki/tree/develop/project/threads/views) folder that I remove and move? You can better understand removing __init__.py operation.

It would be better, if you use `request change` instead of adding comment. 😄  You can request more changes even if you've already made onei

**Comment by AbhijithGanesh** [comment]
Understood Gorkem

**Comment by gorkemarslan** [comment]
@AbhijithGanesh, I don't know whether you are aware of the fact that when you press `@` button on your keyword, Github automatically suggests names for you. After pressing `@`, if you press `g`, you see my user name. In case you don't know, I just want to remind you. 😄

**Comment by gorkemarslan** [comment]
@AbhijithGanesh, do you want me to make any further changes to this PR? To be honest, I realized the issues when I tried to help with issue #1128. After merging, I'll give help for that PR. 🚀

**Comment by AbhijithGanesh** [comment]
Not for this PR, please go ahead and help with #1128, I think I'll need to fetch upstream and check about the issue #1134

**Comment by gorkemarslan** [comment]
@AbhijithGanesh, I forgot to mention that #1128 needs to change `views.py` file but it is wrongly placed in `threads/views` folder. In this PR, I remove `threads/views` folder and update `threads/views.py` file. 

> I realized the issues when I tried to help with issue #1128. After merging, I'll give help for that PR. 🚀

After your approval, I'll be able to merge this PR.

**Review by brylie — APPROVED** [review]

**Pull request merged**